d2l-tspan.html

```diff
@@ -61,6 +67,21 @@
 					value: false,
 					reflectToAttribute: true,
 					observer: '_selectedChanged'
+				},
+				_boundFocusOutHandler: {
+					type: Function,
+					value: function() {
+						return this._focusOutHandler.bind(this);
+					}
+				},
+				customBorder: {
+					type: Boolean,
+					value: false,
+					reflectToAttribute: true,
+				},
+				feedbackInFocus: {
```

Comment:
It looks like this property is only meant to be used internally.  In such cases, the property should be named with `_` prefix to indicate that it is private.  This helps future devs understand the intended scope, but also, some of the doc generation tools use this convention to know which properties are public vs private.